crates/playground/.travis.yml

```diff
@@ -0,0 +1,69 @@
+language: rust
```

Comment:
As discussed today with @achimcc and @nmauersberg , we can drop the .travis.yml file as we're using GitHub Actions for CI.